js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+loginTimeoutHelp=Maximale Zeit, die ein Benutzer zum Abschließen einer Anmeldung benötigt. Es wird empfohlen, diese Zeit relativ lang zu wählen, z. B. 30 Minuten oder mehr.
+loginActionTimeout=Maximaldauer eines Anmeldeaktion
+loginActionTimeoutHelp=Max. Zeit, die ein Benutzer hat, um anmeldebezogene Aktionen wie Passwort aktualisieren oder TOTP konfigurieren abzuschließen. Es wird empfohlen, diesen Wert relativ lang zu wählen, z. B. 5 Minuten oder mehr.
+defaultSigAlg=Standard Signatur Algorithmus
+defaultSigAlgHelp=Standardalgorithmus für das Signieren von Token für den Realm
+oAuthDeviceCodeLifespan=OAuth 2.0 Device Code Lebensdauer
+oAuthDeviceCodeLifespanHelp=Maximale Zeit bis zum Ablauf des Gerätecodes und des Benutzercodes. Dieser Wert muss lang genug sein, um nutzbar zu sein (so dass der Nutzer sein Zweitgerät abrufen, zur Verifizierungs-URI navigieren, sich anmelden usw.), sollte aber kurz genug sein, um die Nutzbarkeit eines für Phishing erlangten Codes zu begrenzen.
+oAuthDevicePollingInterval=OAuth 2.0 Device Abfrageintervall
+oAuthDevicePollingIntervalHelp=Die Mindestzeit in Sekunden, die der Client zwischen Abfrageanforderungen an den Token-Endpunkt warten soll.
+shortVerificationUri=Kurze verification_uri im "Device Authorization flow"
+shortVerificationUriTooltipHelp=Falls festgelegt, wird dieser Wert als verification_uri im "Device Authorization flow" zurückgegeben. Diese Uri muss auf {server-root}/realms/{realm}/device umleiten
+parRequestUriLifespan=Lebensdauer des Request URI für Pushed Authorization Request
+parRequestUriLifespanHelp=Zahl, die die Lebensdauer des Anfrage-URIs angibt.  Der Standardwert ist 1 Minute.
+revokeRefreshToken=Refresh Token widerrufen
+revokeRefreshTokenHelp=Wenn diese Option aktiviert ist, kann ein Refresh-Token nur bis zur maximalen Wiederverwendung des Refresh-Tokens verwendet werden und wird widerrufen, wenn ein anderer Token verwendet wird. Andernfalls werden Refresh-Tokens bei Verwendung nicht widerrufen und können mehrfach verwendet werden.
+accessTokenLifespan=Access Token Lebensdauer
+accessTokenLifespanHelp=Maximale Zeit, bevor ein "Access Token" abläuft. Es wird empfohlen, diesen Wert relativ zur SSO Sitzungdauer kurz zu halten.
+recommendedSsoTimeout=Es wird empfohlen, dass dieser Wert kürzer ist als die Leerlaufzeit der SSO-Sitzung\: {{time}}
+accessTokenLifespanImplicitFlow=Access Token Lebensdauer für Implicit Flow
+clientLoginTimeout=Maximaldauer der Clientanmeldung
+clientLoginTimeoutHelp=Maximale Zeit, die ein Client hat, um das Access-Token-Protokoll zu beenden. Dies sollte normalerweise 1 Minute betragen.
+userInitiatedActionLifespan=Benutzer initiierte Aktionslebensdauer
+userInitiatedActionLifespanHelp=Maximale Zeit, bevor eine von einem Benutzer gesendete Aktionserlaubnis (z. B. eine E-Mail mit dem Hinweis "Passwort vergessen") abläuft. Es wird empfohlen, diesen Wert kurz zu wählen, da erwartet wird, dass der Benutzer schnell auf eine selbst erstellte Aktion reagiert.
```

Comment:
"Aktionserlaubnis" or "Aktionsgenehmigung"?

Reply:
Perhaps "von einem Benutzer angeforderte Aktion" is a better translation here?

Reply:
How about:
```
userInitiatedActionLifespanHelp=Maximale Zeit, bevor eine von einem Benutzer angeforderte Aktion (z. B. eine E-Mail mit dem Hinweis "Passwort vergessen") abläuft. Es wird empfohlen, diesen Wert kurz zu wählen, da erwartet wird, dass der Benutzer schnell auf eine selbst erstellte Aktion reagiert.
```